package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@formatjs/intl-pluralrules": "1.3.9",
     "@formatjs/intl-relativetimeformat": "4.5.1",
+    "@types/classnames": "^2.2.6",
```

Comment:
Looks like this is not needed.  Could you please confirm?

Reply:
It is needed, we're using the `classNames` library in `center_channel.tsx`

Reply:
Thanks @devinbinnie.  Then, my only request is to have it on fix version
```suggestion
    "@types/classnames": "2.2.6",
```

Reply:
Agreed.